lib/tessel/commands.js

```diff
@@ -88,3 +88,15 @@ module.exports.sysupgrade = function(path) {
 module.exports.getMemoryInfo = function() {
   return ['cat', '/proc/meminfo'];
 };
+module.exports.setAccessPointSSID = function(ssid) {
+  return ['uci', 'set', 'wireless.@wifi-iface[1].ssid=' + ssid];
+};
+module.exports.setAccessPointPassword = function(password) {
+  return ['uci', 'set', 'wireless.@wifi-iface[1].key=' + password];
+};
+module.exports.setAccessPointSecurity = function(security) {
+  return ['uci', 'set', 'wireless.@wifi-iface[1].encryption=' + security];
+};
+module.exports.turnAccessPointOn = function(enabled) {
```

Comment:
I was using the `wifi` commands as a reference for naming, i.e. [`turnOnWifi`](https://github.com/tessel/t2-cli/pull/405/files#diff-ccb4fa979baa5d5d841b51a98779dfc9R55). I do like the clear naming connections to the commands being executed on the OpenWRT platform, so I'm open to changing them for ease of use.